src/ccache.cpp

```diff
@@ -1881,6 +1881,8 @@ set_up_context(Context& ctx, int argc, const char* const* argv)
   ctx.orig_args = Args::from_argv(argc, argv);
   ctx.ignore_header_paths = Util::split_into_strings(
     ctx.config.ignore_headers_in_manifest(), PATH_DELIM);
+  ctx.ignore_options = Util::split_into_strings(
+      ctx.config.ignore_options(), " ");
```

Comment:
Check for max one \'\*\' char? More precisely check no char but the last one is a '*'?

Reply:
I was on the fence about using matches at this point, and I was shopping for ideas here. If we like the trailing-star match thing, I can make the check for one and only star at the end part of a setter or something.